executor/logger/worker.go

```diff
@@ -35,12 +36,22 @@ func NewWorker(id int, workQueue chan LogRequest, log logr.Logger, sdepName stri
 
 	var producer *kafka.Producer
 	var err error
+	sslKafka := util.GetSslElements()
```

Comment:
:upside_down_face: Nit, but for good practice this should be within the block `if util.GetKafkaSecurityProtocol() == "SSL" {`

Likewise in the Kafka server